logs/logger.go

```diff
@@ -0,0 +1,10 @@
+package logs
+
+type Logger interface {
+	Log(opts Options, msg string, fields ...Field)
```

Comment:
How should this Enabled(level) be used in SDK? I'm not sure it is needed because logger design is already almost zero-cost for disabled levels. Checking if level is enabled in traces will make the traces much more complicated, while most (good) adapters are already checking it and guaranteeing almost zero-cost for disabled logs.

It is not a problem to add this method to the Logger iface, I'm just not sure why.

Reply:
For example topic API can receive messages in a batch, it can contains many small messages per one network package (up to 50-60 messages).
For 10Mbit/second stream it will up to 35 000 messages, for 100Mbit/second - up to 350 000 messages.

If customer will receive messages one by one - near zero cost may be visible. It can cause some records for every message in max level (start wait message, receive message + may be some internal states), but it will be filtered in most cases be log level filter.

I think tracker com check Enabled if need (on very hot path or if prepare have some cost). The tracer may check or may not check the level.

Logger adapter:
1. Can implement enabled check (return true - for stub).
2. Can be called even if enabled return false or not called at all.

It is way for optimization only.

Reply:
Added `Enabled()` method.